### PR TITLE
Surgery toolset

### PR DESCRIPTION
**Pull request by Kmc2000**
Adds in a surgical version of the integrated toolset, fully working!

when emagged, it will dispense a chloral loaded syringe to put victims to bed before you "operate"

#### Changelog

:cl:
rscadd: Adds in a surgical toolset implant, carry all your surgical tools as you go!
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
do we really need an emag function for anything that resembles a machine

**Comment by Kmc2000** [comment]
The toolset one has one, why not give this surgical one a function to help them traitor tots

**Comment by Super3222** [comment]
> do we really need an emag function for anything that resembles a machine

to answer your question. yes

**Comment by ghost** [comment]
if you want emags to become less used for 'm'illegal hacking functions' add another item to the game that isn't traitor specific yet retardedly common to unlock illegal functions that could be a cool idea.

**Comment by ShadowDeath6** [comment]
screwdrive an emag to get a hacked emag

**Comment by ParagonPrime** [comment]
emag an emag for ULTRAMAG

ultramag + ultramag spawns supermatter sword

best idea

**Comment by ShadowDeath6** [comment]
just multitool the game to add hardmode and if you lose it stabs you in the dick for like 20 brute and if you win you get more prizes

File: code/modules/surgery/organs/augments_arms.dm [hunk 1]
```diff
@@ -180,3 +180,22 @@
...
+	origin_tech = "materials=3;engineering=2;biotech=4"
+	contents = newlist(/obj/item/weapon/retractor, /obj/item/weapon/hemostat, /obj/item/weapon/cautery,
+		/obj/item/weapon/surgicaldrill, /obj/item/weapon/scalpel, /obj/item/weapon/circular_saw, /obj/item/weapon/surgical_drapes)
+
+/obj/item/organ/cyberimp/arm/surgical/l/zone = "l_arm"
+
+/obj/item/organ/cyberimp/arm/surgical/emag_act()
+	if(!(locate(/obj/item/weapon/kitchen/knife/combat/cyborg) in items_list))
+		usr << "<span class='notice'>You unlock [src]'s patient pacification functions!</span>"
+		items_list += new /obj/item/weapon/reagent_containers/syringe/lethal/choral(src)
```
Comment (Cruix) [inline code/modules/surgery/organs/augments_arms.dm:198]: Are you sure the if statement should not be if(!(locate/obj/item/weapon/reagent_containers/syringe/lethal/choral) in items_list)) ?

Comment (Kmc2000) [reply code/modules/surgery/organs/augments_arms.dm:198]: Why the hell is this in there still....oops

**Comment by silicons** [comment]
Correct me if I'm wrong, but are you sure the injector will regenerate chloral, or is that a feature for it to not regenerate?

File: code/modules/surgery/organs/augments_arms.dm (hunk 1 above)
Comment (Cruix) [reply code/modules/surgery/organs/augments_arms.dm:198]: KMC it is still here fix it so I can merge it

Comment (Cruix) [reply code/modules/surgery/organs/augments_arms.dm:198]: Still not fixed.

Comment (X-TheDark) [reply code/modules/surgery/organs/augments_arms.dm:198]: Fix this so that this can be merged.

**Comment by Kmc2000** [comment]
@Cruix 

if(!(locate/obj/item/weapon/reagent_containers/syringe/lethal/choral) in items_list))

**Comment by X-TheDark** [comment]
`if(!(locate(/obj/item/weapon/reagent_containers/syringe/lethal/choral) in items_list))`
or
`var/obj/item/weapon/reagent_containers/syringe/lethal/chloral/thing = locate() in items_list`
`if(!thing)`

File: code/modules/surgery/organs/augments_arms.dm [hunk 2]
```diff
@@ -180,3 +180,22 @@
...
+	name = "S.U.R.G.E.R.Y 5000 toolset"
+	desc = "A device which holds multiple tools used for surgery, the SURGERY 5000 series comes with rapid release tool action."
+	origin_tech = "materials=3;engineering=2;biotech=4"
+	contents = newlist(/obj/item/weapon/retractor, /obj/item/weapon/hemostat, /obj/item/weapon/cautery,
+		/obj/item/weapon/surgicaldrill, /obj/item/weapon/scalpel, /obj/item/weapon/circular_saw, /obj/item/weapon/surgical_drapes)
+
+/obj/item/organ/cyberimp/arm/surgical/l/zone = "l_arm"
+
+/obj/item/organ/cyberimp/arm/surgical/emag_act()
+        if(!(locate/obj/item/weapon/reagent_containers/syringe/lethal/choral) in items_list))
```
Comment (Cruix) [inline code/modules/surgery/organs/augments_arms.dm:196]: Remove the last closing parenthesis

**Comment by Super3222** [comment]
@Kmc2000 Are you still working on this? If not, please close.

**Pull request closed without merging**